# HG changeset patch
# User Mike Miller <mtmiller@octave.org>
# Date 1546895237 28800
#      Mon Jan 07 13:07:17 2019 -0800
# Branch stable
# Node ID 597fd034d2125692634e3792166065312700ccc5
# Parent  4eb1f6e413610c78450e96542553933a95c62438
restore default history file timestamp (bug #55329)

* oct-hist.cc (history_system::history_system): Set the default history
timestamp format string.

diff --git a/libinterp/corefcn/oct-hist.cc b/libinterp/corefcn/oct-hist.cc
--- a/libinterp/corefcn/oct-hist.cc
+++ b/libinterp/corefcn/oct-hist.cc
@@ -252,17 +252,17 @@ namespace octave
   static void
   unlink_cleanup (const char *file)
   {
     octave_unlink_wrapper (file);
   }
 
   history_system::history_system (interpreter& interp)
     : m_interpreter (interp), m_input_from_tmp_file (false),
-      m_timestamp_format_string ()
+      m_timestamp_format_string (default_timestamp_format ())
   { }
 
   void history_system::initialize (bool read_history_file)
   {
     command_history::initialize (read_history_file, default_file (),
                                  default_size (),
                                  sys::env::getenv ("OCTAVE_HISTCONTROL"));
 
